soccer/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(ROBOCUP_LIB_SRC
     "motion/TrapezoidalMotion.cpp"
     "NewRefereeModule.cpp"
     "planning/CompositePath.cpp"
-    "planning/DirectTargetPathPlanner.cpp"
+    #"planning/DirectTargetPathPlanner.cpp"
```

Comment:
Then we have to implement a DirectTargetCommand, Planner and stuff. We can do it but everything else needs to be merged together first. I'd really prefer not to keep pivot working right now because its really screwy right now in motionControl since we don't have a pivotKick pathplanner or really support for angle control in our pathplanning layer yet.